core/trino-main/src/main/java/io/trino/metadata/Metadata.java

```diff
@@ -16,6 +16,7 @@
 import com.google.common.util.concurrent.ListenableFuture;
```

Comment:
commit message is too long (70 letters max)

Also add high level description of what's happening

Reply:
Done. Let me know what you think